src/config.rs

```diff
@@ -50,6 +50,8 @@ const ZTUNNEL_WORKER_THREADS: &str = "ZTUNNEL_WORKER_THREADS";
 const ENABLE_ORIG_SRC: &str = "ENABLE_ORIG_SRC";
 const PROXY_CONFIG: &str = "PROXY_CONFIG";
 
+const UNSTABLE_ENABLE_SOCKS5: &str = "UNSTABLE_ENABLE_SOCKS5";
```

Comment:
```suggestion
const ENABLE_SOCKS5: &str = "ENABLE_SOCKS5";
```

would prefer to not use the UNSTABLE_ prefix, env vars do not have stability guarantees anyway, nor does anyone setting them by hand on their own ztunnel instance.